netif.pyx

```diff
@@ -1119,6 +1119,19 @@ cdef class LaggInterface(NetworkInterface):
 
 
 cdef class BridgeInterface(NetworkInterface):
+
+    member_flags = {
+        'learn': defs.IFBIF_LEARNING,
+        'discover': defs.IFBIF_DISCOVER,
+        'stp': defs.IFBIF_STP,
+        'sticky': defs.IFBIF_STICKY,
+        'edge': defs.IFBIF_BSTP_EDGE,
+        'autoedge': defs.IFBIF_BSTP_AUTOEDGE,
+        'ptp': defs.IFBIF_BSTP_PTP,
+        'autoptp': defs.IFBIF_BSTP_AUTOPTP,
+        'private': defs.IFBIF_PRIVATE
+    }
```

Comment:
We should follow the standard of the library and have enum for this, like `InterfaceFlags`.

Thoughts?

Reply:
Agreed